templates/miq-template.yaml

```diff
@@ -446,6 +524,8 @@ objects:
             value: "${POSTGRESQL_MAX_CONNECTIONS}"
           - name: POSTGRESQL_SHARED_BUFFERS
             value: "${POSTGRESQL_SHARED_BUFFERS}"
+          - name: POSTGRESQL_CONFIG_DIR
```

Comment:
@bdunne This allows more flexibility, as user could potentially change the default location on template and we will process accordingly via image scripts.